airflow/api_fastapi/execution_api/routes/xcoms.py

```diff
@@ -184,13 +218,16 @@ def set_xcom(
     return {"message": "XCom successfully set"}
 
 
-def has_xcom_access(xcom_key: str, token: TIToken) -> bool:
+def has_xcom_access(
+    dag_id: str, run_id: str, task_id: str, xcom_key: str, token: TIToken, write: bool = False
```

Comment:
Nothing using it yet, but I did add the idea of read vs write access here.

Reply:
Cool!

Reply:
Can we make it `write_access: bool = False` to be in line with variables? Or vice verse is fine too